src/badges.coffee

```diff
@@ -25,6 +25,7 @@ badgeList = [
   'founder',
   'guardian1', 'guardian2', 'guardian3', 'guardian4', 'guardian5',
   'hacker1', 'hacker2', 'hacker3', 'hacker4', 'hacker5',
+  'hankjohnson',
```

Comment:
For "oliver-lynton-wolfe" we used dashes between the names, should we use the same convention for other names?